kore/src/Kore/Log/WarnSimplificationWithRemainder.hs

```diff
@@ -46,10 +51,49 @@ data WarnSimplificationWithRemainder =
     WarnSimplificationWithRemainder
         { inputPattern :: TermLike Variable
         , sideCondition :: SideCondition Variable
-        , results :: OrPattern Variable
-        , remainders :: OrPattern Variable
+        , results :: Results
+        , remainders :: Remainders
         }
     deriving Typeable
+    deriving (GHC.Generic)
+
+newtype Results = Results { getResults :: OrPattern Variable }
+
+instance Unparse Results where
+    unparse = unparse . toTermLike . getResults
+    unparse2 = unparse2 . toTermLike . getResults
+
+instance SQL.Column Results where
+    defineColumn = SQL.defineTextColumn
+    toColumn = SQL.toColumn . Pretty.renderText . Pretty.layoutOneLine . unparse
+
+newtype Remainders = Remainders { getRemainders :: OrPattern Variable }
+
+instance Unparse Remainders where
+    unparse = unparse . toTermLike . getRemainders
+    unparse2 = unparse2 . toTermLike . getRemainders
+
+instance SQL.Column Remainders where
+    defineColumn = SQL.defineTextColumn
+    toColumn = SQL.toColumn . Pretty.renderText . Pretty.layoutOneLine . unparse
+
+toTermLike :: OrPattern Variable -> TermLike Variable
```

Comment:
Why not use https://github.com/kframework/kore/blob/b6522cdd9153d35f8c7caac8d28088cbc56d9635/kore/src/Kore/Internal/OrPattern.hs#L168?